### PR TITLE
Classes refactor

### PR DESCRIPTION
**Pull request by herokudev**
### Classes refactor:

- Use JavaScript classes instead of objects and arrays. 
 
- Create class methods to add and remove books.

### CODE REVIEW

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @herokudev, 

Good job on refactoring your code to use classes, it looks great, nice job with the linters and the README file 🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] Unfortunately, I couldn't check the functionality of your app, please, double-check everything and verify if something gets lost when you changed your code, the Add function doesn't work, therefore I couldn't check the delete function.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

believe in change

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi Team 👋🏽, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- The issue from the previous is still not fix

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ class Library {
     const newBooks = JSON.parse(localStorage.getItem('booksList'));
```
Comment (carlos-ssh) [inline main.js:15]: - you are missing scripts in your package.json I can't run your project locally.
- In your README file, provide steps on how to install and run the project locally.
- In your localstorage you have an issue that isn't reflecting the changes that i made after remove items from my list. I have 3 on my list and in the session storage just one item.

**Review by carlos-ssh — APPROVED** [review]
Hi @KingsleyMcSimon and @herokudev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

It looks like everything got resolved here, so I have nothing else to say other than good luck with your future projects!

Keep rocking 🚀!

**Pull request merged**